docs/howto/jython.txt

```diff
@@ -1,74 +1,11 @@
-========================
-Running Django on Jython
-========================
+================
+Django on Jython
+================
 
 .. index:: Jython, Java, JVM
 
 Jython_ is an implementation of Python that runs on the Java platform (JVM).
-This document will get you up and running with Django on top of Jython.
+Django used to run on top of Jython when it still supported Python 2. As
```

Comment:
Suggested wording:
If you want to use Jython (which supports only Python 2.7 at the time of this
writing), you must use Django 1.11.x (the last series to support Python 2).
Jython support may be readded to Django if a Python 3 compatible Jython is 
released.